pom.xml

```diff
@@ -2100,8 +2100,6 @@
             <configuration>
               <targetClasses>
                 <param>com.puppycrawl.tools.checkstyle.api.*</param>
-                <param>com.puppycrawl.tools.checkstyle.filefilters.*</param>
-                <param>com.puppycrawl.tools.checkstyle.filters.*</param>
```

Comment:
I can't comment on the specific line, but shouldn't `targetTests` also remove `filters` and `filefilters`?
Please verify if any other target tests can be removed without affection mutation coverage.